languages/go/config.json

```diff
@@ -177,13 +176,13 @@
         "slug": "slices",
         "uuid": "8bc0c6f6-1891-4688-9ed7-a63e3702f655",
         "concepts": ["slices"],
-        "prerequisites": ["arrays", "numbers"]
+        "prerequisites": ["basics", "iteration", "numbers"]
```

Comment:
`arrays` is the technical foundation of `slices` in Go. But that is pretty advanced stuff and I'd just skip `arrays` and teach `slices` instead as that is what most people will use -- especially beginners.

We can have and `advanced-slices` exercise later teaching about the inner workings of slices and that would have `arrays` as a prerequisite.

(My opinion)

Reply:
Iteration is mostly in there as a qrerequisite so we can use for loops to iterate over the slice. `range` iteration is introduced after `slices` and `maps`. --> Not sure if this is optimal. Open to better suggestions.